native/src/components/PoiFiltersModal.tsx

```diff
@@ -111,13 +111,13 @@ const PoiFiltersModal = ({
             <SortingHint>{t('alphabetLetters')}</SortingHint>
           </Row>
           <TileRow>
-            {poiCategories.map(it => (
+            {poiCategories.map(item => (
               <StyledToggleButton
-                key={it.id}
-                text={it.name}
-                active={it.id === selectedPoiCategory?.id}
-                onPress={() => setSelectedPoiCategory(it.id === selectedPoiCategory?.id ? null : it)}
-                Icon={<SvgUri uri={it.icon} />}
+                key={item.id}
+                text={item.name}
+                active={item.id === selectedPoiCategory?.id}
+                onPress={() => setSelectedPoiCategory(item.id === selectedPoiCategory?.id ? null : item)}
+                Icon={<SvgUri uri={item.icon} />}
```

Comment:
@LeandraH told me its better for readability 🙂

Reply:
Ohh, sorry about that then! We might have to discuss this again how to name vars in these cases :D The previous naming is probably from me based on kotlin naming conventions because I like the lean and short syntax :)